modules/openshift-developer-cli-overwriting-a-mirror-registry.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// *modules/openshift-developer-cli-creating-and-pushing-a-component-from-client-os-to-the-disconnected-cluster
```

Comment:
Replace with: // cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster